docs/aws-sagemaker-hosted-scorer.md

```diff
@@ -0,0 +1,152 @@
+# Driverless AI Deployment Template for AWS Sagemaker Hosted Scorer
+
+This [template](https://github.com/h2oai/dai-deployment-templates/tree/master/aws-sagemaker-hosted-scorer) scores realtime data using Driverless AI Mojo pipeline with Java Runtime and plugs into the AWS SageMaker workflow documented [here](https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms-inference-code.html)
+
+It is a REST API which accepts one data point at a time for prediction in real-time in the hosted SageMaker environment. The user needs to provide Driverless AI license key and  model's  pipeline.mojo file for scoring, see [deploy](###deploy-to-sageMaker) section for details. The versions of software used to create the template like mojo runtime are listed [here](https://github.com/h2oai/dai-deployment-templates/blob/master/gradle.properties#L8).
+
+
+## Overview
+
+### Build Image
+
+Generation of the Docker image is plugged into the build process of this project.
+Run the following command in the root project directory to run the `build` process.
+
+```bash
+./gradlew :aws-sagemaker-hosted-scorer:jibDockerBuild
+```
+
+Verify that the Docker image was created, and take note of the version created.
+
+```bash
+docker images --format "{{.Repository}} \t {{.Tag}}" | grep "h2oai/sagemaker-hosted-scorer"
```

Comment:
@nkpng2k @orendain do you know if we publish this image into docker hub?

Reply:
Yes, the `sagemaker-hosted-scorer`, `google-cloud-run-scorer` and `rest-scorer` all seem to be published to Docker hub.  Originally, we only purposefully published the `rest-scorer`, but it looks like the others may also be caught by one of the [steps in our Jenkins process](https://github.com/h2oai/dai-deployment-templates/blob/master/Jenkinsfile#L186).